agent/api/container/container.go

```diff
@@ -93,6 +93,8 @@ type HealthStatus struct {
 type Container struct {
 	// Name is the name of the container specified in the task definition
 	Name string
+	// DependsOn is the field which specifies the ordering for container startup and shutdown.
+	DependsOn []DependsOn `json:"dependsOn"`
```

Comment:
this needs state file version update.

Reply:
I will update the state file version as part of next PR

Reply:
Actually, might be a good idea to lump them together to prevent branching from incomplete state (like between these two commits).

Also, is "dependsOn" not required? of so, add omitEmpty.